install/kubernetes/helm/meshery/README.md

```diff
@@ -16,10 +16,10 @@ Meshery chart for deploying Meshery
 |-----|------|---------|-------------|
 | affinity | object | `{}` |  |
 | annotations | object | `{}` |  |
-| env.ADAPTER_URLS | string | `"meshery-istio:10000 meshery-linkerd:10001 meshery-consul:10002 meshery-kuma:10007 meshery-osm:10009 meshery-nginx-sm:10010 meshery-nsm:10004 meshery-app-mesh:10005 meshery-traefik-mesh:10006 meshery-cilium:10012"` |  |
+| env.ADAPTER_URLS | string | `"meshery-istio:10000 meshery-linkerd:10001 meshery-consul:10002 meshery-kuma:10007 meshery-nginx-sm:10010 meshery-nsm:10004 meshery-app-mesh:10005 meshery-traefik-mesh:10006 meshery-cilium:10012"` | Optional, Meshery Server adapters with which to pre-configure Meshery Server. |
 | env.EVENT | string | `"mesheryLocal"` |  |
```

Comment:
@MUzairS15 do we need a GH issue to remove this antiquated env var or is it still used?

Reply:
No, it is not required anymore. // @theBeginner86